data/sponsors.json

```diff
@@ -1,23 +1,44 @@
 [
-  {
-      "category_name": "",
-      "sponsors": [
-          {
-              "name":"Jetbrains",
-              "link": "https://www.jetbrains.com/es-es/",
-              "logo": "jetbrains_DM7wGjl.png"   
-          }
-      ]
-  },
-  {
-      "category_name": "Venue",
-      "sponsors": [
-          {
-              "name":"Universidad de Especialidades Espíritu Santo",
-              "link": "https://uees.edu.ec",
-              "logo": "Logo-UEES-2018---Vertical.png"   
-          }
-      ]
-  }
-
+    {
+        "category_name": "Sponsors Gold",
+        "category_icon": "gold_medal.png",
```

Comment:
I don't think the medals fit with the devfest style but if no one else thinks the same go ahead

Reply:
If you have a better suggestion, youre welcome :). WIthout any icon/medal there, this section looked a bit poor.

Reply:
![image](https://github.com/user-attachments/assets/c1e9b958-8e98-4f1f-84ca-78b2cf677af6)

What do you think?

Reply:
Yees it looks better, thanks!!

Reply:
I just pushed a commit, any feedback is welcome :)